.expeditor/config.yml

```diff
@@ -73,13 +73,9 @@ github:
 
 release_branches:
   - main:
+      version_constraint: 5.*
+  - Inspec-4:
```

Comment:
Decided to name it Inspec-4 so did the same changes all over where it's required.

Reply:
How about`inspec-4-stable`? That might also provide us some more clarity over the state of the branch. Example: It could be `stable`, `pre` etc too in future maybe?
Also I feel maybe the branch name should be lowercase, not capitalised as a good practice :)

Reply:
@Vasu1105 How aboutinspec-4-stable? That might also provide us some more clarity over the state of the branch. Example: It could be stable, pre etc too in future maybe?
And I feel the branch name should be lowercase, not capitalised as a good practice :)

Reply:
I just followed what Chef is following https://github.com/chef/chef/blob/main/.expeditor/config.yml#L62 I can lower the case. I am not sure whether we will have something like stable or pre etc. but it's just a matter of branch name here.